src/xdg.rs

```diff
@@ -17,13 +17,17 @@ const PLACEHOLDER_ICON_NAME: &str = "tzompantli-placeholder";
 
 /// Icon lookup paths in reverse order relative to the `$XDG_DATA_DIR`.
 const ICON_PATHS: &[(&str, &str)] = &[
+    ("pixmaps/", "png"),
+    ("icons/hicolor/16x16/apps/", "png"),
+    ("icons/hicolor/24x24/apps/", "png"),
     ("icons/hicolor/32x32/apps/", "png"),
-    ("icons/hicolor/64x64/apps/", "png"),
+    ("icons/hicolor/48x48/apps/", "png"),
+    ("icons/hicolor/512x512/apps/", "png"),
     ("icons/hicolor/256x256/apps/", "png"),
-    ("icons/hicolor/scalable/apps/", "svg"),
     ("icons/hicolor/128x128/apps/", "png"),
     ("pixmaps/", "svg"),
-    ("pixmaps/", "png"),
+    ("icons/hicolor/scalable/apps/", "svg"),
+    ("icons/hicolor/64x64/apps/", "png"),
 ];
```

Comment:
I haven't reordered this much, but I've made two significant changes:
 - Prefer 64x64 PNGs over SVGs — I believe this should just be better for performance since we don't need to render SVGs?
 - Prefer `icons/` over `pixmaps/` — The spec says this should be the case, it's probably best to follow it.